common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java

```diff
@@ -171,17 +184,21 @@ public void onReceivedSslError(final WebView view,
                                    final SslErrorHandler handler,
                                    final SslError error) {
         // Developer does not have option to control this for now
+        final String methodTag = TAG + ":onReceivedSslError";
         super.onReceivedSslError(view, handler, error);
```

Comment:
I don't see 'cancel' being called anymore in the flight-off scenario.  But it seems like tests are validating that it is.  Is the job getting done `super` method invocation?

Reply:
yes, super calls it hence removed.